addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -1074,29 +849,11 @@ QUnit.test('open 2 different chat windows: enough screen width [REQUIRE FOCUS]',
 QUnit.test('open 2 chat windows: check shift operations are available', async function (assert) {
     assert.expect(9);
 
-    const channel = {
-        channel_type: "channel",
-        id: 20,
-        is_pinned: true,
-        name: "General",
-    };
-    const chat = {
-        channel_type: "chat",
-        direct_partner: [{
-            id: 7,
-            name: "Demo",
-        }],
-        id: 10,
-        is_pinned: true,
-    };
-    this.data['mail.channel'].records = [channel, chat];
-    Object.assign(this.data.initMessaging, {
-        channel_slots: {
-            channel_channel: [channel],
-            channel_direct_message: [chat],
-        },
-    });
+    // 2 channels are expected to be found in the messaging menu
+    // only their existence matters, data are irrelevant
+    this.data['mail.channel'].records.push({}, {});
```

Comment:
doesn't look right to have "empty" records in data...

Reply:
Same reasoning here, it's just `create` with default data. Only required fields that don't have default/compute have to be provided. And no fields are required in the mock server. Could argue that `name` should be required for channel if we want to exactly follow the Python, but that's about it.